modules/k8s_eks_addons/variables.tf

```diff
@@ -1,3 +1,13 @@
+variable "addon_context" {
+  description = "AWS and EKS metadata"
+  type        = any
+}
+
+variable "infrastructurename" {
```

Comment:
Would this variable be needed for the future addons?
Why do we need it to be its own variable?

I'd place the variable inside the addon_context.

Reply:
Not sure would it be needed, but still it is good idea to put it into addon_context.

Reply:
Done.